samples/customABSample/src/main/java/com/inmobi/customabsample/CustomABActivity.java

```diff
@@ -0,0 +1,89 @@
+package com.inmobi.customabsample;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.Toast;
+
+import com.inmobi.sdk.InMobiSdk;
+import com.inmobi.sdk.SdkInitializationListener;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class CustomABActivity extends AppCompatActivity {
+
+    private static final String TAG = CustomABActivity.class.getName();
+    private Button bannerIntegration;
+    private Button interstitialIntegration;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        InMobiSdk.setLogLevel(InMobiSdk.LogLevel.DEBUG);
+        JSONObject consent = new JSONObject();
+        try {
+            // Provide correct consent value to sdk which is obtained by User
+            consent.put(InMobiSdk.IM_GDPR_CONSENT_AVAILABLE, true);
```

Comment:
If we are adding sample for consent object then why not IAB string as well?

Reply:
We don't know what IAB VALUE to flow through this API.